restorers/model/nafnet/nafblock.py

```diff
@@ -0,0 +1,90 @@
+import tensorflow as tf
+from tensorflow import keras
+
+
+class SimpleGate(keras.layers.Layer):
```

Comment:
This layer can be simplified into something like the following since the diagram in the paper also refers only to splitting halfway

```
def simple_gate(features):
    x1, x2 = tf.split(features, num_or_size_splits=2, axis=-1)
    return x1 * x2
```